### PR TITLE
FS-2052-set-lang-arg-params

### PR DESCRIPTION
**Pull request by ramsharma-prog**

### CODE REVIEW

**Comment by gidsg** [comment]
Needs to bump the version number

**Comment by ramsharma-prog** [comment]
> Needs to bump the version number

done! thanks

File: fsd_utils/locale_selector/get_lang.py [hunk 1]
```diff
@@ -4,7 +4,16 @@
 
 
 def get_lang():
-    # get locale from cookie if set
+    
+    # get lang if lang query arg is set
+    language_from_query_args = request.args.get("lang")
+    if language_from_query_args:
+        if language_from_query_args not in ["cy", "en"]:
+            raise ValueError("Invalid language code. Supported codes are 'cy' and 'en'.")
```
Comment (gidsg) [inline fsd_utils/locale_selector/get_lang.py:12]: Can we test how the apps work without this condition locally? If they default to `en` it'd be fine to remove this.

Comment (ramsharma-prog) [reply fsd_utils/locale_selector/get_lang.py:12]: Tested as suggested. I guess we need value error for now.

**Review by gio-karunakaran — APPROVED** [review]
LGTM

**Pull request merged**